extensions/functions_set.yaml

```diff
@@ -0,0 +1,89 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: "is_in"
+    description: >
+      Checks the membership of x in values
+
+      Returns True if found, False if not
+    impls:
+      - args:
+          - value: i8
+            name: "x"
+          - value: i8
+            name: "values"
+        return: bool
+      - args:
+          - value: i16
+            name: "x"
+          - value: i16
+            name: "values"
+        return: bool
+      - args:
+          - value: i32
+            name: "x"
+          - value: i32
+            name: "values"
+        return: bool
+      - args:
+          - value: i64
+            name: "x"
+          - value: i64
+            name: "values"
+        return: bool
+      - args:
+          - value: fp32
+            name: "x"
+          - value: fp32
+            name: "values"
+        return: bool
+      - args:
+          - value: fp64
+            name: "x"
+          - value: fp64
+            name: "values"
+        return: bool
+  -
+    name: "index_in"
```

Comment:
I'd expect these to either be variadic or have an argument that is a list of items. I'd also tend to think these should be defined generically e.g. 

```
- args: 
  - value: T
  - value: List<T>
```

Reply:
This looks to be resolved.